src/wp-includes/media.php

```diff
@@ -1963,6 +1976,59 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds 'auto' to the sizes attribute to the image, if the image is lazy loaded and does not already include it.
+ *
+ * @since 6.7.0
+ *
+ * @param string $image The image tag markup being filtered.
+ * @return string The filtered image tag markup.
+ */
+function wp_img_tag_add_auto_sizes( $image ) {
```

Comment:
Another minor suggestion:

```suggestion
function wp_img_tag_add_auto_sizes( string $image ): string {
```

PHP type hints are being used in other new functions added to core, for example `wp_register_script_module()`, `register_block_bindings_source()`, `wp_register_font_collection()`, etc.